scripts/ci-e2e.sh

```diff
@@ -29,8 +29,8 @@ source "${REPO_ROOT}/scripts/environment.sh"
 # Clone dev-env repo
 sudo mkdir -p ${WORKING_DIR}
 sudo chown "${USER}":"${USER}" ${WORKING_DIR}
-M3_DEV_ENV_REPO="https://github.com/metal3-io/metal3-dev-env.git"
-M3_DEV_ENV_BRANCH=main
+M3_DEV_ENV_REPO="https://github.com/Nordix/metal3-dev-env.git"
+M3_DEV_ENV_BRANCH=capm3/v1.6-kashif
```

Comment:
@kashifest can we revert this, otherwise this LGTM

/hold

Reply:
Done, dev-env has its own PR here https://github.com/metal3-io/metal3-dev-env/pull/1321, but I guess we have this chicken and egg issue again that one of them would fail for the other.